charts/kubewarden-controller/templates/_helpers.tpl

```diff
@@ -39,7 +39,11 @@ helm.sh/chart: {{ include "kubewarden-controller.chart" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
 {{- end }}
+app.kubernetes.io/component: policy-controller
```

Comment:
How does this sound?

```suggestion
app.kubernetes.io/component: controller
```

I think it would be more fitting if it would be just `controller`, or `kubewarden-controller`. The controller takes care of `PolicyServer` Deployments in addition to `ClusterAdmissionPolicies` and `AdmissionPolicies` webhooks and so. And in the future it may take care of even more CRDs.

We have also other components, like the [policy-server](https://github.com/kubewarden/policy-server) as part of kubewarden-defaults chart, and the upcoming [audit-scanner](https://github.com/kubewarden/audit-scanner).